cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -27,8 +26,20 @@ <h2 class="header-slug">
         </span>
     </h2>
     {% endif %}
-    {#% TODO: Add condition to display various related post types %#}
-    {{ _related_posts_list() }}
+    {% set related_post_types = page.related_posts %}
+    {% if related_post_types | length  > 1 %}
+    {% for post_type, post_dict in related_post_types.iteritems() %}
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/4f7be3aa7e05110a901bc37d3381b519b3ca0d84